src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int ZERO_INDEX = 0;
+    static final int FIRST_INDEX = 1;
+    static final String CSV_SEPARATOR = ",";
```

Comment:
Where is **private** access modifier? 
![image](https://user-images.githubusercontent.com/20326858/211284641-ed96d6c7-fdbd-462a-8935-fbaf89c2c64e.png)

ZERO_INDEX and FIRST_INDEX are unclear names. Be more attentive reading our comments
![image](https://user-images.githubusercontent.com/20326858/211285004-9338888f-de50-4712-8fc4-1a3fc59ac261.png)